src/kata2.booleanToWord.js

```diff
@@ -1,5 +1,9 @@
 const booleanToWord = (boolean) => {
```

Comment:
If you'd like something some succinct try  a ternary operator like `return Boolean(boolean) ?  'Yes' : 'No'`